src/Console/Command/DescribeCommand.php

```diff
@@ -93,21 +129,48 @@ private function describeRectorCodeSamples(array $codeSamples): void
             $codeAfter .= $codeSample->getCodeAfter() . $separator;
         }
 
-        $formattedDiff = $this->differAndFormatter->bareDiffAndFormat($codeBefore, $codeAfter);
-        if ($formattedDiff) {
-            $this->consoleStyle->write($formattedDiff);
+        return [$codeBefore, $codeAfter];
+    }
+
+    private function printWithCliFormat(int $i, InputInterface $input, RectorInterface $rector): void
+    {
+        $this->consoleStyle->section(sprintf('%d) %s', $i, get_class($rector)));
+
+        $rectorDefinition = $rector->getDefinition();
+        if ($rectorDefinition->getDescription()) {
+            $this->consoleStyle->writeln(' * ' . $rectorDefinition->getDescription());
+        }
+
+        if (! $input->getOption(Option::OPTION_NO_DIFFS)) {
+            $this->describeRectorCodeSamples($rectorDefinition->getCodeSamples());
         }
+
+        $this->consoleStyle->newLine(2);
```

Comment:
Maybe my complaining about unnecessary lines can be solved here

Reply:
Of course, I'm used to use 2 lines before new headline, but I can change to standard 1